app/src/main/res/layout-land/completed_story_list_fragment.xml

```diff
@@ -52,6 +52,8 @@
         android:layout_height="match_parent"
         android:clipToPadding="false"
         android:overScrollMode="never"
+        android:paddingStart="56dp"
+        android:paddingEnd="56dp"
         android:paddingTop="28dp"
         android:paddingBottom="108dp"
```

Comment:
This should be `88dp`

Reply:
it is 100dp in the mock

Reply:
Yeah that is correct. Somehow I was subtracting `12dp`. But it be `92dp`.